### DISCUSSION TITLE
How to not showing the first paragraph of the post under the "blog" part?

### DISCUSSION THREAD

**Discussion by mendax1234** [academicpages/academicpages.github.io#2106] [Q&A] [answered]
I find that the first paragraph of my blog will always be shown under the "blog post" section. But I don't want that. I just want the name of my blog and its creating date. How can I modify to achieve this?
![image](https://github.com/academicpages/academicpages.github.io/assets/62934294/867353a9-0102-4b1d-a637-bb3faf9a95d7)
Thanks!

**Comment by rjzupkoii** [comment, marked as answer]
I'll have to double check later, but I think you can disable the post excerpt by editing `_includes/archive-single.html`, but since blog posts don't have the same excerpt tag that the other ones have it's unclear exactly where that's being pulled in from.

**Reply by mendax1234** [reply]
Thanks! It works!
